wormhole-foundation/wormhole-connect
# Modify build process to add cjs files to the packaged lib

Split the ts build process into two, one for cjs files, and the other for esm.

The way I found to reproduce the issue was the following:
1. Create a test project with `npx create-next-app wh-connect-next`
2. `cd wh-connect-next` and `npm install @wormhole-foundation/wormhole-connect`
3. Replace the contents in `src/pages/index.tsx` with the following
```tsx
import WormholeConnect from '@wormhole-foundation/wormhole-connect'

export default function Home() {
  return (
    <WormholeConnect config={{ env: 'testnet' }} />
  );
}
```
4. Run `npm run dev`, access the url, and see that the error shows up

And to test the solution:
1. Check out this branch (`bugfix/519-add-cjs-build`)
2. `npm run build`
3. Link the local package to the `wh-connect-next` using your preferred method (`yalc`/`npm link`/`npm package`+`npm install`)
4. Run the test project, access the url, and see that the component now loads

Closes #519, #457
